target/linux/mpc85xx/files/arch/powerpc/platforms/85xx/ecw7210.c

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: GPL-2.0+
+
+/*
+ * edgecore ecw7210 Board Setup
+ *
+ * Copyright (C) 2019 Scott Gilliland <zeroping@gmail.com>
+ *
+ * Based on:
+ *   mpc85xx_rdb.c:
+ *      MPC85xx RDB Board Setup
+ *      Copyright 2013 Freescale Semiconductor Inc.
+ *
+ * This program is free software; you can redistribute  it and/or modify it
+ * under  the terms of  the GNU General  Public License as published by the
+ * Free Software Foundation;  either version 2 of the  License, or (at your
+ * option) any later version.
+ */
+
+#include <linux/stddef.h>
+#include <linux/kernel.h>
+#include <linux/delay.h>
+#include <linux/interrupt.h>
+#include <linux/of_platform.h>
+
+#include <asm/time.h>
+#include <asm/machdep.h>
+#include <asm/pci-bridge.h>
+#include <mm/mmu_decl.h>
+#include <asm/prom.h>
+#include <asm/udbg.h>
+#include <asm/mpic.h>
+
+#include <sysdev/fsl_soc.h>
+#include <sysdev/fsl_pci.h>
+#include "smp.h"
+
+#include "mpc85xx.h"
+
+void __init ecw7210_pic_init(void)
+{
+	struct mpic *mpic;
+
+	mpic = mpic_alloc(NULL, 0,
+	  MPIC_BIG_ENDIAN |
+	  MPIC_SINGLE_DEST_CPU,
+	  0, 256, " OpenPIC  ");
+
+	BUG_ON(mpic == NULL);
+	mpic_init(mpic);
+
+}
+
+/*
+ * Setup the architecture
+ */
+static void __init ecw7210_setup_arch(void)
+{
+	if (ppc_md.progress)
+		ppc_md.progress("ecw7210_setup_arch()", 0);
+
+	mpc85xx_smp_init();
+
+	fsl_pci_assign_primary();
+
+        printk(KERN_INFO "ECW7210 board from Edge-core\n");
```

Comment:
As before, please take care of your indent before submitting (applies to whole file as well).